### PR TITLE
Add routes to FE

### PR DESCRIPTION
**Pull request by GeorgDV**
Changes from #4

---

This adds 3 routes for the applications:
- Home (main page)
- About
- Error (currently works as 404 page)

Also this adds a startup for component library hierarchy. Currently works as such as suggested by @Kasparsu:
```
components/
--layouts/
--pages/
---Home/
-----Home.tsx
-----index.ts
---About/
-----About.tsx
-----index.ts
---Error/
-----Error.tsx
-----index.ts
index.tsx
App.tsx
```

### CODE REVIEW

**Review by Kasparsu — CHANGES_REQUESTED** [review]
App.tsx should be in src as this is entrypoint of the app. its also in the hierarchy it might be bit confusing but its on same level as coponents folder. Move that back to src.

File: fe/src/components/pages/About/About.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+function About(): JSX.Element {
+  // @TODO - Add some actual content here.
+  return (
+    <div>
+      <h1>About Page</h1>
+    </div>
+  );
+}
+
+export default About;
```
Comment (Orzelius) [inline fe/src/components/pages/About/About.tsx:12]: We should look into [eol-last](https://eslint.org/docs/rules/eol-last)

**Comment by Kasparsu** [comment]
> App.tsx should be in src as this is entrypoint of the app. its also in the hierarchy it might be bit confusing but its on same level as coponents folder. Move that back to src.

not only tsx but all the things relating to the App.

**Review by Kasparsu — CHANGES_REQUESTED** [review]
also maybe move `logo.svg` to assets folder can be in `src/assets` maybe.

**Comment by GeorgDV** [comment]
> also maybe move `logo.svg` to assets folder can be in `src/assets` maybe.

Yep, some folder like `src/assets` makes sense for such files

**Review by Orzelius — APPROVED** [review]
This is a new type of structure for a react app (for me) but I'm coming to like it.

**Comment by Kasparsu** [comment]
> This is a new type of structure for a react app (for me) but I'm coming to like it.

maybe take a look at nuxt.js. I know its vue but there are bunch of cool things that could be transfered to react projects. I personally like the folder structrure there in essence.

**Review by Kasparsu — APPROVED** [review]
All good! Squash and merge.

File: fe/src/components/pages/About/About.tsx (hunk 1 above)
Comment (GeorgDV) [reply fe/src/components/pages/About/About.tsx:12]: Sure 👍

**Pull request merged**